### PR TITLE
feat(button): add light type support

### PR DESCRIPTION
**Pull request by unix**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

### CODE REVIEW

File: pages/en-us/components/button.mdx [hunk 1]
```diff
@@ -50,15 +50,19 @@ Used to trigger an operation.
...
-  <Button type="error">Error</Button> <Spacer y={.5} />
-  <Button type="abort">Abort</Button>
-</>
+<Grid.Container gap={2}>
+  <Grid><Button auto type="secondary">Secondary</Button></Grid>
+  <Grid><Button auto type="success">Success</Button></Grid>
+  <Grid><Button auto type="warning">Warning</Button></Grid>
+  <Grid><Button auto type="error">Error</Button></Grid>
+  <Grid><Button auto type="abort">Abort</Button></Grid>
+  <Grid><Button auto type="secondary-light">Secondary Ligth</Button></Grid>
```
Comment (ofekashery) [inline pages/en-us/components/button.mdx:61]: It should be ``Light``.

Comment (unix) [reply pages/en-us/components/button.mdx:61]: Now the dilemma is that `light` can only be combined with 4 basic type.(`success-light` / `warning-light` ...) 
If `light` is a single prop, but it only works under 4 basic type, which seems strange 🤕

**Comment by unix** [comment]
After adding the `light` type, the `Button` component becomes heavier 😢

File: pages/en-us/components/button.mdx (hunk 1 above)
Comment (ofekashery) [reply pages/en-us/components/button.mdx:61]: I mean that you have a typo in the button texts (lines 61-64) - it should be ``Light``, not ``Ligth``.

Comment (unix) [reply pages/en-us/components/button.mdx:61]: 😿

**Comment by unix** [comment]
I created new branch named `rc` on origin, all `pre-release` versions are easier to release  from `rc`.

**Pull request closed without merging**